### DISCUSSION TITLE
Feature Request: Compact Sidebar Contents

### DISCUSSION THREAD

**Discussion by kyleshrader** [ztjhz/BetterChatGPT#166] [Ideas]
This is a request for modifying the design of the sidebar's contents.

Currently the contents of the sidebar has a lot of margin/padding on icons and text that makes it harder to read. A design with less padding, no folder/chat icon, and reduced size on the edit/delete icons would be more readable to some (me).

This is just an idea, if it isn't a common want then it's probably not worth the effort.

**Comment by ztjhz** [comment]
Let's see what other think about this.

We could also potentially implement a compact view option that you can toggle in the settings.

**Comment by ztjhz** [comment]
@kyleshrader We have made the sidebar more compact!
